openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -170,34 +172,68 @@ def test_send_update_emails_to_registrants_in_webinar_update(webinar_admin_insta
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('update', (True, False))
-def test_save_related_send_emails(request, webinar_admin_instance, webinar, mocker, update):
+@pytest.mark.parametrize('is_update, is_already_published, is_set_to_be_published', [
+    (False, False, False),
+    (False, False, True),
+    (False, True, False),
```

Comment:
You are right that this is not a real scenario, a published webinar cannot be unpublished. I just added this to test the code just in case. I'll remove it.